tests/test_tileserver.py

```diff
@@ -91,3 +92,22 @@ def test_create_with_dict(sample_svs):
         response = client.get("/layer/Test/zoomify/TileGroup0/0-0-0.jpg")
         assert response.status_code == 200
         assert response.content_type == "image/jpeg"
+
+
+def test_cli_name_multiple_flag():
+    """Test cli_name multiple flag."""
+    opt = cli_name()
```

Comment:
Any particular reason this is being renamed? It could be confusing to see `@opt` if this was missed higher up.

Reply:
No, it was just a relic of something else I tried. Have removed the renaming.